src/app/components/profile-edit/profile-edit.service.ts

```diff
@@ -62,4 +64,27 @@ export class ProfileEditService implements OnDestroy {
                 );
         }
     }
+    setUrl(url: string, deletePreviousPath: string): any {
+        this.deletePreviousPath = deletePreviousPath;
+        return (this.newUrl = url);
```

Comment:
bad practice to return the assignment
set return type as void and don't return at all

Reply:
So i need just assign:     _this.newUrl = url_ and don't use return. 
But in some case i had a lot of troubles when i had forgotten use _return_ in some fuctions.

Reply:
because of _any_ type